### PR TITLE
Fix #30

### LINKED ISSUE #30: Elmish debugger can't work with decimal
[ELMISH DEBUGGER] Cannot generate auto encoder for System.Decimal. Please pass an extra encoder.

I think this error is rooted at:

image

### PR DESCRIPTION
**Pull request by alfonsogarciacaro**
- Add extra coders (decimal and longs, do we need bigint too?)
- Add option to pass customs Thoth coders

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+### 3.0.1
```
Comment (et1975) [inline RELEASE_NOTES.md:5]: Need to bump this?

File: src/debugger.fs [hunk 2]
```diff
@@ -44,28 +47,31 @@ module Debugger =
...
-            let inflate x =
-                match Decode.fromValue "$" decoder x with
-                | Ok x -> x
-                | Error er -> failwith er
-            deflate, inflate
+            let coders =
+                Extra.empty
+                |> Extra.withDecimal
+                |> Extra.withInt64
+                |> Extra.withInt64
```
Comment (vbfox) [inline src/debugger.fs]: Twice Int64 for luck ?

File: src/debugger.fs [hunk 3]
```diff
@@ -44,28 +47,31 @@ module Debugger =
...
+                // |> Extra.withBigInt
```
Comment (vbfox) [inline src/debugger.fs:69]: True, it's in separate lib, I'm not sure dragging bigint is worth it by default

**Review by vbfox — APPROVED** [review]

File: src/debugger.fs (hunk 2 above)
Comment (alfonsogarciacaro) [reply src/debugger.fs]: Ups, should be the unsigned version. I'll fix it, thanks!

File: src/debugger.fs (hunk 3 above)
Comment (alfonsogarciacaro) [reply src/debugger.fs:69]: Well, decimals and longs are also in a separate lib now, but they're used much more frequently so...

**Pull request merged**